pkg/runtime/runtime.go

```diff
@@ -401,15 +411,36 @@ func (m *MosnRuntime) initStates(factorys ...*runtime_state.Factory) error {
 	return nil
 }
 
+func (m *MosnRuntime) initOss(oss ...*oss.OssFactory) error {
+	log.DefaultLogger.Infof("[runtime] init file service")
+
+	// register all oss store services implementation
+	m.ossRegistry.Register(oss...)
+	for name, config := range m.runtimeConfig.Oss {
+		c, err := m.ossRegistry.Create(config.Type)
+		if err != nil {
+			m.errInt(err, "create oss component %s failed", config.Type)
+			return err
+		}
+		if err := c.InitConfig(context.TODO(), &config); err != nil {
+			m.errInt(err, "init oss component %s failed", config.Type)
+			return err
+		}
+		m.oss[name] = c
+	}
+	return nil
+}
+
 func (m *MosnRuntime) initFiles(files ...*file.FileFactory) error {
 	log.DefaultLogger.Infof("[runtime] init file service")
 
 	// register all files store services implementation
 	m.fileRegistry.Register(files...)
+
 	for name, config := range m.runtimeConfig.Files {
 		c, err := m.fileRegistry.Create(config.Type)
 		if err != nil {
-			m.errInt(err, "create files component %s failed", name)
+			m.errInt(err, "create files component %s failed", config.Type)
```

Comment:
Why change it from `name` to `config.Type`?
All types of components use `name` here:
![image](https://user-images.githubusercontent.com/26001097/179888502-4ccb25cc-9e34-4607-af55-c8eb76fa970d.png)

Reply:
done